library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -490,8 +492,16 @@ impl ExitStatus {
         libc::WIFEXITED(self.0)
     }
 
-    pub fn success(&self) -> bool {
-        self.code() == Some(0)
```

Comment:
does this just straight up remove the `success` method or am I reading this wrong?